manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
You can use 'Refs #issue' instead of 'Fixes #issue' to have the PR tracked,but not marking the issue as fixed.
The advantage is also it might get into the master sooner than the rest of this changes (that I've just made you to be dependent on https://github.com/theforeman/puppet-foreman_proxy/pull/140 :) I guess other capsule features also need this workaround to start working properly again, right?